clang/lib/Sema/SemaChecking.cpp

```diff
@@ -19800,7 +19808,8 @@ bool Sema::SemaBuiltinVectorMath(CallExpr *TheCall, QualType &Res) {
   return false;
 }
 
-bool Sema::SemaBuiltinElementwiseTernaryMath(CallExpr *TheCall) {
+bool Sema::SemaBuiltinElementwiseTernaryMath(CallExpr *TheCall,
+                                             bool enforceFloatingPointCheck) {
```

Comment:
Formal arguments are capitalized in LLVM's style, like "EnforceFloatingPointCheck". I'm also not a huge fan of this name - it isn't obvious what it means without looking at the implementation of the function.

Reply:
renamed to `CheckForFloatArgs`